command/agent/testagent.go

```diff
@@ -223,7 +223,8 @@ RETRY:
 
 func (a *TestAgent) start() (*Agent, error) {
 	if a.LogOutput == nil {
-		a.LogOutput = testlog.NewWriter(a.T)
+		prefix := fmt.Sprintf("%v:%v ", a.Config.BindAddr, a.Config.Ports.RPC)
```

Comment:
Why is the prefix the address rather than the node ID?

Reply:
I didn't fine a better ID - Server uses the RPC port to identify the node (in raft v2), and the client UUID isn't always available (many tests only has the server component enabled).  Given that many server logs already include rpc port, it makes sense to use that as the prefix to ease corrolation.

`a.Config.NodeName` seems non-unique and primarily represent the test name in many of the tests.